### PR TITLE
Update documentation links

### PR DESCRIPTION
**Pull request by KevinFCormier**
Addresses documentation link updates from:
https://issues.redhat.com/browse/ACM-15043
https://issues.redhat.com/browse/ACM-15021

### CODE REVIEW

**Review by zlayne — DISMISSED** [review]

File: frontend/src/lib/doc-util.tsx [hunk 1]
```diff
@@ -51,16 +51,19 @@ export const DOC_LINKS = {
...
-  HYPERSHIFT_MANAGE_KUBEVIRT: `${DOC_BASE_PATH}/clusters/index#hosted-control-planes-manage-kubevirt`,
-  HOSTED_ENABLE_FEATURE_AWS: `${DOC_BASE_PATH}/clusters/index#enable-or-disable-hosted-control-planes`,
-  HYPERSHIFT_OIDC: `${DOC_BASE_PATH}/clusters/index#hosted-create-aws-secret`,
-  GITOPS_CONFIG: `${DOC_BASE_PATH}/applications/index#gitops-overview`,
+  GITOPS_CONFIG: `${DOC_BASE_PATH}/gitops/gitops-overview`,
   DEPRECATIONS_AND_REMOVALS: `${DOC_BASE_PATH}/release_notes/release-notes#deprecations-removals`,
   ENABLE_OBSERVABILITY: `${DOC_BASE_PATH}/observability/observing-environments-intro#enabling-observability-service`,
   ACCESSING_CONSOLE: `${DOC_BASE_PATH}/web_console/web-console#accessing-your-console`,
-  ROSA_DISCOVERY_AUTOIMPORT_POLICY_EXAMPLE: `${DOC_BASE_PATH}/clusters/cluster_mce_overview#import-discover-rosa`,
+  ROSA_DISCOVERY_AUTOIMPORT_POLICY_EXAMPLE: `${DOC_BASE_PATH}/multicluster_engine_operator_with_red_hat_advanced_cluster_management/mce-acm-integration#import-discover-rosa`,
```
Comment (dockerymick) [inline frontend/src/lib/doc-util.tsx:58]:
```suggestion
  ROSA_DISCOVERY_AUTOIMPORT_POLICY_EXAMPLE: `${DOC_BASE_PATH}/mce_acm_integration/acm_integrate_import_rosa#import-discover-rosa`,
```

File: frontend/src/lib/doc-util.tsx [hunk 2]
```diff
@@ -6,10 +6,13 @@ import { TextContent } from '@patternfly/react-core'
 import { useTranslation } from './acm-i18next'
 
 export const DOC_VERSION = '2.13'
```
Comment (dockerymick) [inline frontend/src/lib/doc-util.tsx:8]: Should the version be 2.12?

Comment (KevinFCormier) [reply frontend/src/lib/doc-util.tsx:8]: This PR is for our next release (we've branched already).

File: frontend/src/lib/doc-util.tsx (hunk 1 above)
Comment (dockerymick) [reply frontend/src/lib/doc-util.tsx:58]: Nvm i see that it works

File: frontend/src/lib/doc-util.tsx (hunk 2 above)
Comment (dockerymick) [reply frontend/src/lib/doc-util.tsx:8]: oh ok then

**Review by dockerymick — APPROVED** [review]

File: frontend/src/lib/doc-util.tsx [hunk 3]
```diff
@@ -51,16 +51,19 @@ export const DOC_LINKS = {
   MANAGE_APPLICATIONS: `${DOC_BASE_PATH}/applications/index#managing-applications`,
   ANSIBLE_JOBS: `${DOC_BASE_PATH}/clusters/index#ansible-config-cluster`,
   POLICIES_OVERVIEW: `${DOC_BASE_PATH}/governance/index#policy-overview`,
```
Comment (dockerymick) [inline frontend/src/lib/doc-util.tsx:53]: Based on my comment here: https://issues.redhat.com/browse/ACM-15043
```suggestion
  POLICIES_OVERVIEW: `${DOC_BASE_PATH}/governance/index#hub-policy-framework`,
```

Comment (dockerymick) [reply frontend/src/lib/doc-util.tsx:53]: Can this also be updated for 2.12?

**Pull request merged**